src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
+  "vulkanmod.options.pages.graphics": "Графика",
+  "vulkanmod.options.pages.optimizations": "Оптимизация",
+  "vulkanmod.options.pages.other": "Другое",
+
+  "vulkanmod.options.buttons.apply": "Применить",
+  "vulkanmod.options.buttons.kofi": "Поддержите меня",
+
+  "vulkanmod.options.advCulling": "Отбраковка чанков",
```

Технически модель местности всё же усекается, а не отбраковывается целиком

```suggestion
  "vulkanmod.options.advCulling": "Усечение чанков",
```